lib/spinjector/runner.rb

```diff
@@ -0,0 +1,32 @@
+# Inject script phases into your Xcode project
+#
+# @author Guillaume Berthier
+#
+
+require 'xcodeproj'
+require_relative './project_service'
+require_relative './yaml_parser'
+
+class Runner
+
+  attr_reader :project
+
+  def initialize(project_path, configuration_file_path, logger)
+    raise "[Error] No xcodeproj found at #{project_path}" unless File.exist?(project_path)
```

Comment:
C’est redondant avec `raise "[Error] No xcodeproj found in #{Dir.pwd}" if project_path.nil?` dans le spinjector.rb non ?

Reply:
Pas tout à fait, parce que dans le spinjector.rb l'erreur est levé pour éviter d'avoir un objet nil. Dans le runner on vérifie juste qu'il y a un fichier derrière le path.